Editor/Unity/DotNetWizard.cs

```diff
@@ -0,0 +1,64 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+ namespace Innoactive.CreatorEditor
+ {
+     /// <summary>
+     /// Modal Window that helps setting the 'API Compatibility Level' to '.Net 4.x'.
+     /// </summary>
+     internal class DotNetWizard : EditorWindow
+     {
+         private bool abortBuilding;
+         private BuildTargetGroup buildTargetGroup;
+         private ApiCompatibilityLevel currentLevel;
+         private readonly Vector2 fixedSize = new Vector2(400f, 110);
+
+         public bool ShouldAbortBuilding()
+         {
+             return abortBuilding;
+         }
+
+         private void OnEnable()
+         {
+             abortBuilding = false;
+             minSize = maxSize = fixedSize;
+             titleContent = new GUIContent("API Compatibility level*");
```

Comment:
```suggestion
             titleContent = new GUIContent("API Compatibility Level*");
```
Also is this * on purpose?

Reply:
We can remove the `*`, I put it there because that is how it is named in the editor:

<img width="540" alt="Screenshot 2020-10-13 at 15 14 14" src="https://user-images.githubusercontent.com/6911992/95865315-d5d0fa80-0d66-11eb-9130-bc4e23c50ea2.png">